src/main/java/book/store/online/controller/OrderController.java

```diff
@@ -0,0 +1,80 @@
+package book.store.online.controller;
+
+import book.store.online.dto.request.PlaceOrderDto;
+import book.store.online.dto.request.UpdateOrderStatusDto;
+import book.store.online.dto.response.OrderDto;
+import book.store.online.dto.response.OrderItemDto;
+import book.store.online.dto.response.OrderWithoutDetailsDto;
+import book.store.online.service.OrderService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.validation.Valid;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Order management",
+        description = "Endpoints for management users orders")
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final OrderService orderService;
+
+    @Operation(summary = "Get orders data",
+            description = "Get all order by passed token for user")
+    @GetMapping
+    @PreAuthorize("hasRole('ROLE_USER')")
+    public List<OrderDto> getUserOrders(HttpServletRequest request) {
```

Comment:
I would recommend using the `SecurityContextHolder.getContext().getAuthentication()`
or `@RequestHeader(HttpHeaders.AUTHORIZATION) String token` or `@AuthenticationPrincipal UserDetails userDetails` instead of `HttpServletRequest`

It's hard to write tests if you pass HttpServletRequest through all services
Please, fix it everywhere.

https://www.baeldung.com/get-user-in-spring-security
https://www.baeldung.com/spring-rest-http-headers